packages/edit-site/src/plugins/index.js

```diff
@@ -54,23 +54,6 @@ registerPlugin( 'edit-site', {
 					>
 						{ __( 'Manage all reusable blocks' ) }
```

Comment:
While we're at it, could we remove this item too (also duplicated)

I wonder what else is defining these now!

Reply:
Wow good catch, I didn't even notice that one :D 

Done:

<img width="353" alt="Screenshot 2020-09-15 at 08 12 28" src="https://user-images.githubusercontent.com/1204802/93172534-57634780-f72b-11ea-95b6-b7df43f6068a.png">